packages/ui/package.json

```diff
@@ -30,8 +30,8 @@
     "start": "vite",
     "build": "tsc && vite build --config vite.config.js",
     "build:lib": "yarn rimraf ./lib && yarn copy-catalog && yarn build:esm && yarn build:cjs",
-    "build:esm": "tsc --project tsconfig.esm.json && copyfiles -u 1 ./src/**/*.scss ./src/assets/** ./lib/esm",
-    "build:cjs": "tsc --project tsconfig.cjs.json && copyfiles -u 1 ./src/**/*.scss ./src/assets/** ./lib/cjs",
+    "build:esm": "tsc --project tsconfig.esm.json && copyfiles -u 1 './src/**/*.scss' './src/assets/**' ./lib/esm",
+    "build:cjs": "tsc --project tsconfig.cjs.json && copyfiles -u 1 './src/**/*.scss' './src/assets/**' ./lib/cjs",
```

Comment:
This is needed for executing commands in PowerShell, without the quotes, the `**` gets expanded creating an error since the resulting line is too long